lib/ast.js

```diff
@@ -282,9 +282,10 @@ var AST_With = DEFNODE("With", "expression", {
 
 /* -----[ scope and functions ]----- */
 
-var AST_Scope = DEFNODE("Scope", "directives variables functions uses_with uses_eval parent_scope enclosed cname", {
+var AST_Scope = DEFNODE("Scope", "is_block directives variables functions uses_with uses_eval parent_scope enclosed cname", {
```

Comment:
Maybe rename `is_block` to `is_block_scope` to prevent confusion about "block statement" vs "block scope"